src/crypto/crypto.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * This file is necessary to ensure that the crypto library is available
+ * in node and the browser
+ */
+
+// eslint-disable-next-line no-restricted-syntax
+import { webcrypto as nodeCrypto } from 'crypto'
```

Comment:
You've tested this in a browser to confirm this won't blow up?

Reply:
i have. created a build and everything seemed to work in the Inbox app.

Reply:
Great stuff.